tensorflow/compiler/tf2xla/functionalize_while.cc

```diff
@@ -200,33 +200,29 @@ Status BuildLoopBody(const Graph& graph, Frame* frame,
     arg_types->push_back(dtype);
 
     TF_ASSIGN_OR_RETURN(Node * arg_node, BuildArgNode(output, dtype, i));
-
-    if (dtype == DT_RESOURCE) {
-      // The convention of the XLA bridge is that resource variable arguments
-      // are only inputs to the loop body and have no corresponding output.
-      // TODO(b/37741920): change the convention so that DT_RESOURCE variables
-      // are both inputs and outputs, and then remove this case.
-      TF_RET_CHECK(arg.is_loop_invariant);
+    TF_ASSIGN_OR_RETURN(Node * retval_node,
```

Comment:
Tip: you could do `TF_ASSIGN_OR_RETURN(Node * const retval_node, ...)` to prevent clang-format from getting confused and adding a whitespace incorrectly.